### DISCUSSION TITLE
Zfs-PVs and full_page_writes=off

### DISCUSSION THREAD

**Discussion by rowa78** [cloudnative-pg/cloudnative-pg#3058] [General]
Hi,

i am testing the cloudnative-pg and i really like it so far. I just want to ask about the possibility to deactivate the full_page_writes if you use zfs-volumes.

At the moment i run the openebs zfs-localpv. If you read the tips about tuning postgresql on zfs, there is often a recommendation to disable the full_page_writes, because zfs is a Copy on Write fs.

For example: https://www.2ndquadrant.com/en/blog/pg-phriday-postgres-zfs/

At the moment, this is not possible with the operator, because this parameter is fixed. (https://cloudnative-pg.io/documentation/1.20/postgresql_conf/)

What do you think? Would this be a good idea or "dangerous"?

**Comment by prehor** [comment]
Same with me. I'm using cloudnative-pg together with OpenEBS zfs-localpv and would like to disable full_page_writes permanently.

According to https://wiki.postgresql.org/wiki/Full_page_writes#When_can_you_turn_it_off.3F this shouldn't be a problem anymore - both ZFS 2.2 and PostgreSQL 15 have been released for a long time.
